agent/handlers/task_server_setup_integ_test.go

```diff
@@ -125,26 +125,26 @@ func TestRateLimiterIntegration(t *testing.T) {
 			name:            "Different network faults A1, A2 + same method B1",
 			method1:         "POST",
 			method2:         "POST",
-			url1:            "/api/container123/fault/v1/network-blackhole-port/status",
```

Comment:
Just curious, was there an issue with using status here?

Reply:
Yes. Since we removed the requirement on request payload for latency/loss fault's status and stop, if we use status here for latency/fault endpoints, the request will actually go through instead of getting blocked due to empty payload, which means we would have to add extra mock expects.

Reply:
I see, thanks for clarifying